### PR TITLE
Minor improvements around formatting and data gathering.

### PR DESCRIPTION
**Pull request by kunalspathak**
- Some bug fixes and add header in res.csv file
- Set highest priority for thread
- Better command usage formatting
- Per thread elapsed time/ticks
- Fix ONELINE_STATS

### CODE REVIEW

**Comment by kunalspathak** [comment]
@maoni0 @mrsharm

File: PrimeNumbers/PrimeNumber_join.cpp [hunk 1]
```diff
@@ -422,25 +435,41 @@ class PrimeNumbers
 
         if (ht_used)
         {
-            ht_used_p = (ht_used == 1);
+            ht_used_p = (ht == 1);
```
Comment (Maoni0) [inline PrimeNumbers/PrimeNumber_join.cpp:437]: thanks for fixing my bug!

File: PrimeNumbers/PrimeNumber_join.cpp [hunk 2]
```diff
@@ -98,7 +101,7 @@ const double _1M = pow(10, 6);
 const double _1K = pow(10, 3);
 const char* formatNumber(double input)
```
Comment (Maoni0) [inline PrimeNumbers/PrimeNumber_join.cpp]: so I think I actually meant to change this arg from double to uint64_t but it got erased when I sync-ed to the latest when submitting that PR. do we actualy need this to be double at all? if not we can just change it to uint64_t

**Review by Maoni0 — APPROVED** [review]
LGTM!

File: PrimeNumbers/PrimeNumber_join.cpp (hunk 2 above)
Comment (kunalspathak) [reply PrimeNumbers/PrimeNumber_join.cpp]: fixed

**Pull request merged**